wiremock/mappings/HmppsAuthAndDelius_GetUser_JimSnow.json

```diff
@@ -2,18 +2,19 @@
   "priority": 2,
   "request": {
     "method": "GET",
-    "urlPattern": "/user/.*"
+    "urlPattern": "/user/JIMSNOWLDAP"
   },
   "response": {
+    "transformers": ["response-template"],
     "status": 200,
     "jsonBody": {
       "userId": 2500077027,
       "username": "JIMSNOWLDAP",
       "firstName": "Jim",
       "surname": "Snow",
-      "email": "test@digital.justice.gov.uk",
+      "email": "jim.snow@example.digital.justice.gov.uk",
       "enabled": true,
-      "roles": ["ROLE_PROBATION", "ROLE_APPROVED_PREMISES_STAFF"]
+      "roles": ["ROLE_PROBATION"]
```

Comment:
Do we want all these users to have the same role?

Reply:
thanks Pat. These roles are from HMPPS-Auth and the only HMPPS-provided role we care about is `ROLE_PROBATION`, which is automatically granted to anyone who authenticates via Delius.

Internal CAS1 roles (e.g. `MANAGER`, `WORKFLOW_MANAGER`) are managed within the service. The UI obtains them with a call to the API at `/profile`.

In this commit I'm removing `ROLE_APPROVED_PREMISES_STAFF` which is cruft. It's not used anywhere and I think was added by mistake.